lib/tasks/report_primary_organisation.rake

```diff
@@ -0,0 +1,79 @@
+namespace :report_primary_organisation  do
+  desc "A one-off CSV report of how many documents were initially published by authors who don't belong to the document's primary organisation"
+  task :generate_csv, [:user_email] => :environment do |t, args|
+    require 'csv'
+
+    class CSVReport
+      def get_authors(edition)
+        edition.publication_audit_entry.edition.authors.first
+      end
+
+      def author_belongs_to_diff_org?(author, organisation)
+        author.organisation.present? && organisation.present? && organisation.id != author.organisation.id
+      end
+
+      def header_row
+        ["An admin link",
+         "Date first published",
+         "Title",
+         "Format",
+         "Lead organisation",
+         "First Author name",
+         "First Author email address",
+         "First Author organisation",
+         "Last Author name",
+         "Last Author email address",
+         "Last Author organisation"]
+      end
+
+      def get_last_author(edition)
+        edition.latest_edition.published_by
+      end
+
+      def filtered_editions
+        Edition.where("state = 'published' AND first_published_at >= '2014-01-01'")
+      end
+
+      def generate_csv
+        CSV.generate do |csv|
+          csv << header_row
+          filtered_editions.first(20).each do |e|
+            puts "Edition: #{e.id}"
+            presenter = DocumentListExportPresenter.new(e)
+
+            first_author = e.creator
+            primary_organisation = if e.type == 'CorporateInformationPage'
```

Comment:
Can you use the `presenter.lead_organisations` method?  It returns a string or an array, but I reckon we could work around that.